addons/ai/functions/fnc_addWaveHandler.sqf

```diff
@@ -1,23 +1,28 @@
+#include "\x\tmf\addons\AI\script_component.hpp"
 /*
  * Name: TMF_ai_fnc_addWaveHandler
  * Author: Head
  *
  * Arguments:
  * 0: The wave spawner logic <OBJECT>
  * 1: Code to execute <CODE>
+ * 2: Global effect <BOOL>
  *
  * Return:
- * None
+ * Nothing
  *
  * Description:
  * Adds an eventhandler to a wave spawner that execute everytime a wave is spawned.
- * Local execution only.
  */
 
-params ["_logic","_code"];
+params ["_logic","_code", ["_isGlobal",false]];
+
+if (_isGlobal && !isRemoteExecuted) exitWith {
+    [_logic,_code,false] remoteExecCall [QFUNC(addWaveHandler),0,true];
+};
 
 private _handlers = _logic getVariable ["Handlers", []];
 private _index = _handlers pushBack _code;
 _logic setVariable ["Handlers", _handlers, false];
 
-_index
+//_index
```

Comment:
Why is it commented out?

Reply:
It lists no return, and might be misleading when using the new isGlobal command.

Reply:
I mean there is a removeWaveHandler and you need the index for it.